krod/core/engine.py

```
@@ -33,10 +34,10 @@
 # from krod.core.decision import DecisionSystem, Decision
 
 # security validator
-from .security_validator import SecurityValidator
+from krod.core.security_validator import SecurityValidator
```

Flake8 (7.2.0)

[error] redefinition of unused 'SecurityValidator' from line 20

(F811)